### PR TITLE
move report, slack and pubsub packages

### PR DESCRIPTION
**Pull request by lwsanty**
close [[DEV-340]]
related to [[DEV-339]]

[DEV-340]: https://athenianco.atlassian.net/browse/DEV-340
[DEV-339]: https://athenianco.atlassian.net/browse/DEV-339

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module github.com/athenianco/cloud-common
+
+go 1.13
+
+require (
+	github.com/athenianco/metadata v0.0.0-20200619130750-022c72de9d66
```
Comment (vmarkovtsev) [inline go.mod:6]: This kind of perverts the whole point of extracting a common library :smile:

Comment (dennwc) [inline go.mod:6]: `common` must not depend on the MD. Let's copy everything that is needed to this package to avoid this dependency.

Comment (lwsanty) [reply go.mod]: you guys are kidding

Comment (lwsanty) [reply go.mod]: done
p.s. when I told that we need to move pubsub you said no

**Comment by lwsanty** [comment]
> Can you please also add `EnsureTimeout` from `funcs.go`
Done.

**Pull request merged**